### PR TITLE
Better documentation

### PR DESCRIPTION
**Pull request by Doyouwanttomakeacake**
There is still many things to do, especially regarding the wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,38 @@
 # DiscordConsole ![License](https://img.shields.io/badge/license-GPL-blue.svg?style=flat-square) [![Support](https://img.shields.io/badge/Discord-Support%20guild-6C88EE.svg?style=flat-square)](https://discord.gg/xvQV8bT)
 
-The console that allows you to control both your user and bot account in interesting ways. 
-[Download Win/Mac/Linux 64-bit binaries](https://github.com/LEGOlord208/DiscordConsole/releases)
+> DiscordConsole is a software authored by LEGOlord208 aiming to give you full control over accounts, bots and weebhooks!
```
Comment (legolord208) [inline README.md]: > authored by LEGOlord208

maybe skip this?

Comment (legolord208) [inline README.md]: Also,
> weebhook

hehe typo

File: README.md [hunk 2]
```diff
@@ -1,56 +1,38 @@
...
-DiscordConsole lets you bulk delete messages. This allows you to delete a bunch of messages at once, without needing to write any code!
+Try to type something like `help` and let the magic happens! For a detail review of each commands, check this wiki page *Soon™* or type `help command`.
 
-### Log
-Log the last 100 messages to a file, or just view the last 10 directly in the console!
+### Contribute
 
-## Command line
-DiscordConsole has full command line support. You can supply a bot/user token or email/password on the command line, and even specify commands to run, so you can use DiscordConsole for scripting.
+Feels free to submit a Pull Request! Not a coder, but want to be involved in the project? Send us some suggestions trough isssues.
```
Comment (legolord208) [inline README.md]: through*

File: README.md [hunk 3]
```diff
@@ -1,56 +1,38 @@
...
 
-Having it start with a specific server automatically selected? Making a cron job to automatically message how many days until Trump leaves? Easy!  
-![Imgur](http://i.imgur.com/2mst4pH.png)  
+### License
+
+GNU G.P.L. © Legolord208 - [Click here for more informations.](https://github.com/legolord208/DiscordConsole/blob/master/LICENSE)
```
Comment (legolord208) [inline README.md]: Small nitpick, but could you keep the casing consistent and put `Lego` in capitals? Thanks!

File: README.md [hunk 4]
```diff
@@ -32,12 +32,12 @@ Try to type something like `help` and let the magic happens! For a detail review
...
+DiscordConsole authored by [LEGOlord208](https://github.com/LEGOlord208), with documentation improvments by [Parallaχ](https://github.com/Doyouwanttomakeacake).
 
 ### Contribute
 
 Feels free to submit a Pull Request! Not a coder, but want to be involved in the project? Send us some suggestions trough isssues.
 
 ### License
 
-GNU G.P.L. © Legolord208 - [Click here for more informations.](https://github.com/legolord208/DiscordConsole/blob/master/LICENSE)
+GNU G.P.L. © Legolord208 - [Click here for more informations.](https://github.com/LEGOlord208/DiscordConsole/blob/master/LICENSE)
```
Comment (legolord208) [inline README.md]: > informations
Typo again. information*

File: README.md [hunk 5]
```diff
@@ -32,12 +32,12 @@ Try to type something like `help` and let the magic happens! For a detail review
 
 ### Credits
 
-DiscordConsole authored by [Legolord208](https://github.com/legolord208) with help from [Tdbot](https://github.com/tbodt), [Mnpn](https://github.com/Mnpn03) and [Parallax](https://github.com/Doyouwanttomakeacake).
+DiscordConsole authored by [LEGOlord208](https://github.com/LEGOlord208), with documentation improvments by [Parallaχ](https://github.com/Doyouwanttomakeacake).
```
Comment (legolord208) [inline README.md]: > authored by
sounds odd...

Could you change it to `written by` or `coded by` or `made by` or whatever?
Again, SORRY for nitpicking!

File: README.md [hunk 6]
```diff
@@ -1,56 +1,43 @@
...
 
-Having it start with a specific server automatically selected? Making a cron job to automatically message how many days until Trump leaves? Easy!  
-![Imgur](http://i.imgur.com/2mst4pH.png)  
+### Contribute
+
+Feels free to submit a Pull Request! Not a coder, but want to be involved in the project? Send us some suggestions trough isssues.
+
+### License
+
+GNU G.P.L. © Legolord208 - [Click here for more information.](https://github.com/LEGOlord208/DiscordConsole/blob/master/LICENSE)
```
Comment (legolord208) [inline README.md]: `LEGOlord208` please :stuck_out_tongue:

File: README.md [hunk 7]
```diff
@@ -1,56 +1,43 @@
...
 
-## Command line
-DiscordConsole has full command line support. You can supply a bot/user token or email/password on the command line, and even specify commands to run, so you can use DiscordConsole for scripting.
+DiscordConsole wrote by [LEGOlord208](https://github.com/LEGOlord208), with documentation improvments by [Parallaχ](https://github.com/Doyouwanttomakeacake).
 
-Having it start with a specific server automatically selected? Making a cron job to automatically message how many days until Trump leaves? Easy!  
-![Imgur](http://i.imgur.com/2mst4pH.png)  
+### Contribute
+
+Feels free to submit a Pull Request! Not a coder, but want to be involved in the project? Send us some suggestions trough isssues.
```
Comment (legolord208) [inline README.md]: > trough

File: README.md [hunk 8]
```diff
@@ -1,56 +1,43 @@
...
-DiscordConsole lets you bulk delete messages. This allows you to delete a bunch of messages at once, without needing to write any code!
+Try to type something like `help` and let the magic happens! For a detail review of each commands, check this wiki page *Soon™* or type `help command`.
 
-### Log
-Log the last 100 messages to a file, or just view the last 10 directly in the console!
+### Credits
 
-## Command line
-DiscordConsole has full command line support. You can supply a bot/user token or email/password on the command line, and even specify commands to run, so you can use DiscordConsole for scripting.
+DiscordConsole wrote by [LEGOlord208](https://github.com/LEGOlord208), with documentation improvments by [Parallaχ](https://github.com/Doyouwanttomakeacake).
```
Comment (legolord208) [inline README.md]: > wrote

written*

**Comment by legolord208** [comment]
Really cool! Thanks :smile:

**Comment by Mnpn03** [comment]
i still offer my work for a "proper" website with things on

**Comment by legolord208** [comment]
@Mnpn03 Please do make pull requests as well :smile:

File: README.md (hunk 1 above)
Comment (Mnpn03) [inline README.md]: weeb-hook lenny

**Pull request merged**